src/standardPronunciation.ts

```diff
@@ -0,0 +1,316 @@
+import { joinString } from './_internal';
+import { isHangulAlphabet, isHangulCharacter } from './_internal/hangul';
+import { combineHangulCharacter } from './combineHangulCharacter';
+import {
+  ㄴㄹ이_덧나는_모음,
+  ㄴㄹ이_덧나서_받침_ㄴ_변환,
+  ㄴㄹ이_덧나서_받침_ㄹ_변환,
+  된소리,
+  된소리_받침_23항,
+  받침_대표음_발음,
+  발음변환_받침_ㅎ,
+  발음변환_받침_ㅎ_발음,
+  발음변환_첫소리_ㅎ,
+  발음변환_첫소리_ㅎ_발음,
+  비음화_받침_ㄴ_변환,
+  비음화_받침_ㅁ_변환,
+  비음화_받침_ㅇ_변환,
+  어간_받침,
+  음가가_없는_자음,
+  음의_동화_받침,
+  자음동화_받침_ㄴ_변환,
+  특별한_한글_자모,
+  특별한_한글_자모의_발음,
+  한글_자모,
+} from './constants';
+import { disassembleCompleteHangulCharacter } from './disassembleCompleteHangulCharacter';
+import { hasProperty, isIncludedInArray, isNotUndefined } from './utils';
+
+type NotHangul = {
+  index: number;
+  syllable: string;
+};
+
+type NonUndefined<T> = T extends undefined ? never : T;
+type Syllable = NonUndefined<ReturnType<typeof disassembleCompleteHangulCharacter>>;
+
+/**
+ * 주어진 한글 문자열을 표준 발음으로 변환합니다.
+ * @param hangul 한글 문자열을 입력합니다.
+ * @param options 변환 옵션을 설정합니다.
+ * @param options.hardConversion 경음화 등의 된소리를 적용할지 여부를 설정합니다. 기본값은 true입니다.
+ * @returns 변환된 표준 발음 문자열을 반환합니다.
+ */
+export function standardPronunciation(
```

Comment:
함수이기 때문에, 명사가 아닌 동사를 사용하는것은 어떤가요~?

<img width="1331" alt="image" src="https://github.com/toss/es-hangul/assets/69495129/4ac594bc-9688-4d26-8ddb-9a0c535cb7c1">
```suggestion
export function standardizePronunciation(
```

Reply:
https://github.com/toss/es-hangul/pull/132#discussion_r1652126733

위의 논의가 끝나고 함수 명명규칙이 더 확고해지면 그때 함수 이름을 변경할게요. 해당 리뷰는 잠시 보류해놓을게요.